### DISCUSSION TITLE
希望可以支持骰子码识别

### DISCUSSION THREAD

**Discussion by liergazi** [Mrs4s/go-cqhttp#706] [Ideas]
酷Q以前的码 [CQ:dice,type=1-6]
现在mirai2.5M支持了识别骰子，群内发送骰子魔法表情 日志可以看到对应的mirai码
[mirai:dice:1-6]
希望go cqhttp也能支持* _ *

**Comment by Mrs4s** [comment]
会考虑支持的

**Reply by liergazi** [reply]
> 会考虑支持的

🙏
